db/migrate/20200617105046_add_new_column_to_accounts_bought_c_instances_count.rb

```diff
@@ -0,0 +1,7 @@
+class AddNewColumnToAccountsBoughtCInstancesCount < ActiveRecord::Migration[5.0]
+  def change
+    safety_assured do
```

Comment:
This shouldn't be necessary for this migration, is it? 👀 Because if it is, we have a problem 😝